core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -197,29 +185,29 @@ class RequestQuotaTest extends BaseRequestTest {
         case ApiKeys.UPDATE_METADATA_KEY =>
           val partitionState = Map(tp -> new PartitionState(Int.MaxValue, brokerId, Int.MaxValue, List(brokerId).asJava, 2, Set(brokerId).asJava)).asJava
           val securityProtocol = SecurityProtocol.PLAINTEXT
-          val brokers = Set(new requests.UpdateMetadataRequest.Broker(brokerId,
-            Seq(new requests.UpdateMetadataRequest.EndPoint("localhost", 0, securityProtocol,
+          val brokers = Set(new UpdateMetadataRequest.Broker(brokerId,
+            Seq(new UpdateMetadataRequest.EndPoint("localhost", 0, securityProtocol,
             ListenerName.forSecurityProtocol(securityProtocol))).asJava, null)).asJava
-          new requests.UpdateMetadataRequest.Builder(ApiKeys.UPDATE_METADATA_KEY.latestVersion, brokerId, Int.MaxValue, partitionState, brokers)
+          new UpdateMetadataRequest.Builder(ApiKeys.UPDATE_METADATA_KEY.latestVersion, brokerId, Int.MaxValue, partitionState, brokers)
 
         case ApiKeys.CONTROLLED_SHUTDOWN_KEY =>
-          new requests.ControlledShutdownRequest.Builder(brokerId)
+          new ControlledShutdownRequest.Builder(brokerId)
 
         case ApiKeys.OFFSET_COMMIT =>
-          new requests.OffsetCommitRequest.Builder("test-group",
-            Map(tp -> new requests.OffsetCommitRequest.PartitionData(0, "metadata")).asJava).
+          new OffsetCommitRequest.Builder("test-group",
+            Map(tp -> new OffsetCommitRequest.PartitionData(0, "metadata")).asJava).
             setMemberId("").setGenerationId(1).setRetentionTime(1000)
 
         case ApiKeys.OFFSET_FETCH =>
-          new requests.OffsetFetchRequest.Builder("test-group", List(tp).asJava)
+          new OffsetFetchRequest.Builder("test-group", List(tp).asJava)
 
         case ApiKeys.FIND_COORDINATOR =>
-          new requests.FindCoordinatorRequest.Builder(FindCoordinatorRequest.CoordinatorType.GROUP, "test-group")
+          new FindCoordinatorRequest.Builder(FindCoordinatorRequest.CoordinatorType.GROUP, "test-group")
 
         case ApiKeys.JOIN_GROUP =>
-          new JoinGroupRequest.Builder("test-join-group", 10000, "", "consumer",
-            List( new JoinGroupRequest.ProtocolMetadata("consumer-range",ByteBuffer.wrap("test".getBytes()))).asJava)
-           .setRebalanceTimeout(60000)
+          new JoinGroupRequest.Builder("test-join-group", 200, "", "consumer",
+            List(new JoinGroupRequest.ProtocolMetadata("consumer-range", ByteBuffer.wrap("test".getBytes()))).asJava)
+           .setRebalanceTimeout(100)
```

Comment:
Do we need to change this if `GroupInitialRebalanceDelayMsProp` is set to 0? The group should rebalance immediately unless another group member joins before `InitialDelayedJoin.onComplete` is called. 
I guess there is no harm in setting it to 100